spdx/parsers/validations.py

```diff
@@ -151,11 +152,7 @@ def validate_doc_spdx_id(value, optional=False):
 def validate_doc_namespace(value, optional=False):
     if value is None:
         return optional
-    elif (
-        value.startswith("http://")
-        or value.startswith("https://")
-        or value.startswith("ftp://")
-    ) and ("#" not in value):
+    elif uritools.isabsuri(value) and ("#" not in value):
         return True
     else:
         return False
```

Comment:
```suggestion
    return uritools.isabsuri(value) and ("#" not in value)
```

I think we could shorten this a bit.

Reply:
Good point. I was reluctant to touch the structure at first, but going through the whole file erasing lines was quite satisfying :D